charts/example-configurations/external-gitlab-values.yaml

```diff
@@ -0,0 +1,55 @@
+# Configuration file for deploying renku using an existing GitLab
+# instance. Examples for gitlab.com and renkulab.io/gitlab are
+# explicitly given.
+# See
+# https://github.com/SwissDataScienceCenter/renku/blob/master/charts/README.rst
+# and
+# https://renku.readthedocs.io/en/latest/developer/example-configurations/external-gitlab)
+# for instructions on how to use this values file.
+
+variables:
+  gitlabURL:          &gitlabURL           # https://gitlab.com or https://renkulab.io/gitlab
+  gitlabClientId:     &gitlabClientId      # Put GitLab Application Id here!
+  gitlabClientSecret: &gitlabClientSecret  # Put GitLab Secret here!
+  gitlabRegistryHost: &gitlabRegistryHost  # registry.gitlab.com or registry.renkulab.io
+
+# Comment out or remove this block when using renkulab.io/gitlab
+global:
+  gitlab:
+    urlPrefix: ""
+
+gitlab:
+  enabled: false
+
+notebooks:
+  gitlab:
+    url: *gitlabURL
+    registry:
+      host: *gitlabRegistryHost
+  jupyterhub:
+    hub:
+      extraEnv:
+        - name: GITLAB_URL
+          value: *gitlabURL
+        - name: DEBUG
+          value: "1"
+        - name: JUPYTERHUB_SPAWNER_CLASS
+          value: spawners.RenkuKubeSpawner
+        - name: PGPASSWORD
+          valueFrom:
+              secretKeyRef:
+                name: renku-jupyterhub-postgres
+                key: jupyterhub-postgres-password
```

Comment:
Are these values needed? Aren't these just the defaults from the notebooks service?

Reply:
We're doing this in all our values files because of helm (2?) not merging lists: https://github.com/helm/helm/issues/3486

Reply:
Since we need the GITLAB_URL as an env variable for Jupyterhub, we have to repeat all other env variables too. Not exactly dry...